BuildAndTest.ps1

```diff
@@ -138,13 +140,13 @@ if (-not $NoPackage) {
     $versionPrefix, $versionSuffix = .\src\build\Get-VersionConstants.ps1
     $version = "$versionPrefix$versionSuffix"
 
-    $packagingProjects = "Json.Pointer", "Json.Schema", "Json.Schema.ToDotNet", "Json.Schema.Validation"
+    $packagingProjects = "Json.Pointer", "Json.Schema"
     foreach ($project in $packagingProjects) {
         New-NuGetPackageFromProjectFile $project $version
     }
 
-    $nuspecProjects = @()
+    $nuspecProjects = "Json.Schema.ToDotNet.Cli", "Json.Schema.Validation.Cli"
```

Comment:
>Cli [](start = 44, length = 3)

I understand why you might want to name the command line tools this way. But I don't agree with naming the _packages_ this way. The name is restrictive, it suggests to me that the package contains _only_ the command line tool. Whereas if you named the packages `Json.Schema.ToDotNet`, `Json.Schema.Validation`, then I would not be surprised to find they contained a command line tool in addition to the library.

Reply:
Oh, I'm sorry! I now see from the `.nuspec` file that the package _is_ named as I suggest. It's just the project file that has this name. This is fine.

---
In reply to: [213830491](https://github.com/Microsoft/jschema/pull/30#discussion_r213830491) [](ancestors = 213830491)